kingpin/actors/base.py

```diff
@@ -308,6 +308,15 @@ def _fill_in_contexts(self, context={}, strict=True):
         # Finally, convert the string back into a dict and store it.
         self._options = json.loads(new_options_string)
 
+    def get_orgchart(self, parent=''):
+        return [
+            [{'v': str(id(self)),
```

Comment:
I think its absolutely time for the BaseActors ` self.__repr__` to return useful and consistent data about themselves. Something like:

```(aws.elb.WaitUntilHealthy) <insert options['desc'] here>```